### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by andronedev**

### CODE REVIEW

**Comment by andronedev** [comment]
1. config the .env file
2. run the ./build.sh command
3. run ./run.sh and enjoy !

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+RUN apt update
+RUN apt install python3-pip -y
+
+RUN mkdir /app
+ADD . /app
+WORKDIR /app
+RUN pip install -r requirements.txt
+
+# tricks for pytube : https://github.com/elebumm/RedditVideoMakerBot/issues/142
```
Comment (CallumIO) [inline Dockerfile:11]: This is not necessary as pytube is no longer a dependency

Comment (andronedev) [reply Dockerfile:11]: oh I see, I will rectify that

**Comment by andronedev** [comment]
for the last commit, I reset because I made a mistake during a push, everything is now good.

**Review by CallumIO — APPROVED** [review]
Looks good, thanks for the contribution!

**Pull request merged**